src/quacc/recipes/emt/core.py

```diff
@@ -89,3 +95,72 @@ def relax_job(
     dyn = run_opt(atoms, relax_cell=relax_cell, **opt_flags)
 
     return summarize_opt_run(dyn, additional_fields={"name": "EMT Relax"})
+
+
+@job
+def microcanonical_job(
+    atoms: Atoms,
+    initial_temperature_params: dict[str, Any] | None = None,
+    md_params: dict[str, Any] | None = None,
+    **calc_kwargs,
+) -> DynSchema:
+    """
+    Carry out a microcanonical ensemble calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    initial_temperature_params
+        Dictionary of custom kwargs for the initial temperature distribution. Set a value
+        to `quacc.Remove` to remove a pre-existing key entirely. For a list of available
+        keys, refer to [ase.md.velocitydistribution.MaxwellBoltzmannDistribution][].
+        Quacc has two additional parameters `fixcm` and `fixrot` to fix the center of mass
+        and rotation, respectively when setting the initial temperature. The default is
+        `{"temperature": None, "fixcm": True, "fixrot": True}`. If `temperature` is set to
+        `None`, the initial temperature will not be set. For an improved change of success when performing a microcanonical ensemble calculation, the initial temperature should be set to a value twice the
+        target temperature.
+    md_params
+        Dictionary of custom kwargs for the optimization process. Set a value
+        to `quacc.Remove` to remove a pre-existing key entirely. For a list of available
+        keys, refer to [quacc.runners.ase.run_md][].
+    **calc_kwargs
+        Custom kwargs for the EMT calculator. Set a value to
+        `quacc.Remove` to remove a pre-existing key entirely. For a list of available
+        keys, refer to the `ase.calculators.emt.EMT` calculator.
+
+    Returns
+    -------
+    DynSchema
+        Dictionary of results, specified in [quacc.schemas.ase.summarize_md_run][].
+        See the type-hint for the data structure.
+    """
+
+    md_defaults = {"timestep": 1.0, "steps": 500, "dynamics": VelocityVerlet}
+
+    initial_temperature_defaults = {"temperature": None, "fixcm": True, "fixrot": True}
+
+    initial_temperature_params = recursive_dict_merge(
+        initial_temperature_defaults, initial_temperature_params
+    )
+
+    temperature = initial_temperature_params.pop("temperature", None)
```

Comment:
In NVE temperature is not well defined, usually what is done is that initial velocities are initialised to a given temperature via a distribution. That is why I didn't put "temperature" directly in the parameter since it could be confusing: "temperature for a micro-canonical job, what is that???"

Same for the comment below, fix_rot or fix_cm are only for the initialisation phase, if the users wants proper COM fixing they should pass an Atoms object with a FixCom constraint.

That's why I put everything in an "initial..." dict, so that the users are clearly aware that this is limited to the initialisation phase. I am open to changing things, just some additional info here

Reply:
>> Are these initial_temperature_params going to be included for all other ensembles as well

Well, for NVT or NPT, temperature plays a key a role, we will have a `temperature` keyword alone. But initial_temperature_params will have to be here as well. Usually the initial temperature should be equal to the one you aim, but everything is possible really.

I thought about adding these parameters individually as you suggest, but ultimately I found that having 4 different only for the initialisation phase is not ideal. I don't have a strong opinion here, both approach seems ok to me, with a slight preference to the dict form.

Reply:
Let's stick with the `dict` form then :)